dags/main_summary.py

```diff
@@ -629,19 +629,19 @@
 search_clients_daily_bigquery = bigquery_etl_query(
     task_id="search_clients_daily_bigquery",
     destination_table="search_clients_daily_v8",
-    dataset_id="search_staging",
+    dataset_id="search",
+    project_id="moz-fx-data-shared-prod",
     owner="bewu@mozilla.com",
     email=["telemetry-alerts@mozilla.com", "bewu@mozilla.com"],
-    sql_file_path='sql/search/search_clients_daily_v8/query.sql',
     dag=dag)
 
 search_aggregates_bigquery = bigquery_etl_query(
     task_id="search_aggregates_bigquery",
     destination_table="search_aggregates_v8",
-    dataset_id="search_staging",
+    dataset_id="search",
```

Comment:
this should write to a `search_derived` dataset, and `search.search_aggregates` and `search.search_clients_daily` should be views.